cristi-lupu/Youth
# Use SDWebImage

Reviewer: cristian.lupu

### Background

`SDWebImage` has better caching system and handling of image downloading than `AlamofireImage`.

### What has been done?
1. Updated loading images to `SDWebImage`.
2. Updated author name to `Cristian Lupu`.

### How to test?
1. Build project.
2. Run unit tests.
3. Check images on fast scrolling.

### Checklist
